obspy/core/inventory/response.py

```diff
@@ -1001,25 +995,24 @@ def get_unit_mapping(key):
                                             max_f))
 
                 amp = scipy.interpolate.InterpolatedUnivariateSpline(
-                    f, amp, k=3)(freqs)
+                    f, amp, k=3)(frequencies)
                 phase = scipy.interpolate.InterpolatedUnivariateSpline(
-                    f, phase, k=3)(freqs)
+                    f, phase, k=3)(frequencies)
 
                 # Set static offset to zero.
                 amp[amp == 0] = 0
                 phase[phase == 0] = 0
 
                 rl = blkt.blkt_info.list
-                rl.nresp = len(freqs)
+                rl.nresp = len(frequencies)
 
                 _freq_c = (C.c_double * rl.nresp)()
                 _amp_c = (C.c_double * rl.nresp)()
                 _phase_c = (C.c_double * rl.nresp)()
 
-                for i in range(len(freqs)):
-                    _freq_c[i] = freqs[i]
-                    _amp_c[i] = amp[i]
-                    _phase_c[i] = phase[i]
+                _freq_c[:] = frequencies[:]
+                _amp_c[:] = amp[:]
+                _phase_c[:] = phase[:]
```

Comment:
Hmm. I cannot believe I did not try this the first time around. Lol - this should also be quite a bit faster now :-)